src/simulation/m_acoustic_src.fpp

```diff
@@ -159,6 +159,7 @@ contains
         integer, parameter :: mass_label = 1, mom_label = 2
 
         sim_time = t_step*dt
+        if (lag_bubbles) sim_time = time_tmp
```

Comment:
I'm not sure I understand what the purpose of this assignment is. Could you elaborate a little?

Reply:
Thank you @wilfonba for your comments. The s_4th_5th_order_rkck time stepper can indeed be used with an adaptive functionality, meaning that the time step (dt) may vary. As a result, the physical time will not necessarily be equal to t_step*dt.

Reply:
Yes, I added two test cases that use the adaptive RKCK stepper. The first one uses a one-way EL coupling approach and the other uses a two-way EL coupling approach. 

Addressing your second question: The key difference lies in the method used to adjust the time step (dt). The existing implementation calculates dt using the CFL number from the Eulerian framework (please correct me if I'm wrong). On the other hand, the adaptive RKCK stepper, calculates dt based on the truncation error of the Lagrange variables between the 4th and 5th order RKCK time stepper solutions.

Reply:
Yes this sounds right to me. So do you have an example in the test suite that uses the 'regular' RK3 (possible adaptive one-way) time stepper (and another that uses RKCK)?

Reply:
Currently, the test cases are only utilizing the RKCK algorithm. Two of the cases employ the adaptive RKCK stepper, while the other two use the RKCK stepper with a constant time step. I am currently working on porting the Lagrange bubbles subroutines into the standard RK1, RK2, and RK3 steppers, which will address one of your comments.

Reply:
perfect, I appreciate it! i think having as much flexibility will be handy

Reply:
update on this would be useful.. was this resolved or are there questions still? I've kind of gotten confused

Reply:
Now the highlighted line command gets activated when 'time_stepper==4' (rkck switch). This command modifies the physical time, which will not necessarily be equal to t_step*dt if the adaptive functionality is activated. 

This is the update for the test cases: 

- Two cases employ the 3rd RK stepper: one of them uses of the one-way EL coupling approach, and the other uses the two-way EL coupling approach
- Two cases uses the adaptive RKCK stepper: one of them makes use of the one-way EL coupling approach, and the other uses the two-way EL coupling approach